cmd/plugins/juju-metadata/validatetoolsmetadata.go

```diff
@@ -45,17 +45,17 @@ type validateToolsMetadataCommand struct {
 
 var validateToolsMetadataDoc = `
 validate-tools loads simplestreams metadata and validates the contents by
-looking for tools belonging to the specified series, architecture, for the
-specified cloud. If version is specified, tools matching the exact specified
+looking for agent binaries belonging to the specified series, architecture, for the
+specified cloud. If version is specified, agent binaries matching the exact specified
 version are found. It is also possible to just specify the major (and optionally
 minor) version numbers to search for.
 
 The cloud specification comes from the current Juju model, as specified in
 the usual way from either the -m option, or JUJU_MODEL. Series, Region, and
 Endpoint are the key attributes.
 
-It is possible to specify a local directory containing tools metadata, in which
-case cloud attributes like provider type, region etc are optional.
+It is possible to specify a local directory containing agent binaries metadata, 
```

Comment:
metadata for agent binaries

Reply:
I've changed most occurrences to 'agent metadata'... I'll ping you once I commit these changes..